kernel/kselftest.py

```diff
@@ -60,7 +60,7 @@ def setUp(self):
             deps = deps + rdeps
         elif 'SuSE' in detected_distro.name:
             deps = deps + sdeps
-        elif 'centos' in detected_distro.name:
+        elif detected_distro.name in ['centos', 'fedora']:
             deps = deps + cdeps
```

Comment:
Ok. Those variable names are not very clear (cdeps, sdeps, ...), but that's a different matter.